packages/nx-phrase/src/lib/utils.ts

```diff
@@ -65,3 +66,24 @@ export function prepareOutput({
 
     return baseOutputPath
 }
+
+export interface LanguageAndRegion {
+    language: string
+    region?: string
+}
+
+export function getLanguageAndRegion(locale: string): LanguageAndRegion {
+    // check if locale is supported, if not it throws an error
+    let [normalizedLocale] = Intl.NumberFormat.supportedLocalesOf(locale)
```

Comment:
I would not recommend a normalization here since in most cases the locales available are downloaded from Phrase anyway and the used function has a fallback logic which might lead to inappropriate results.